FCC1.html

```diff
@@ -0,0 +1,106 @@
+<<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title id="title">
+    Dr. Norman Borlaug
+  </title>
+<style type="text/css">
```

Comment:
It would be better if you write your CSS codes in separate file to increase performance.

Reply:
Great job being polite, phrasing your comment as a suggestion, and giving a reason for the requested change